modules/standard.js

```diff
@@ -37,21 +38,24 @@ class Standard{
         return embed;
     }
 
-    handleMessage(command, parameter, msg) {
-        /*if(this.cachedEmbed != null && this.cachedTime != null && new Date().getTime()-this.cachedTime<this.cacheExpireTime){
-            return msg.channel.send('',{embed: this.cachedEmbed});
-        }*/
+    loadList(){
         rp({url: this.api, json:true}).then(body=>{
-         return msg.channel.send('',{embed: this.generateEmbed(body)});
+            return this.generateEmbed(body);
         },err=>{
             log.error("Error getting Standard list",err.error.details);
-            return msg.channel.send('',{embed: new Discord.RichEmbed({
+            return new Discord.RichEmbed({
                 title: "Standard - Error",
                 description: "Couldn't create Standard list.",
                 color: 0xff0000
-            })});
+            });
         });
+    }
 
+    handleMessage(command, parameter, msg) {
+        if(this.cachedEmbed != null && this.cachedTime != null && new Date().getTime()-this.cachedTime<this.cacheExpireTime){
+            return msg.channel.send('',{embed: this.cachedEmbed});
+        }
+        return msg.channel.send('',{embed: this.loadList()});
```

Comment:
`this.loadList()` doesn't return an embed, I think. It *should* return a promise (due to the request being asynchronous) and when it does, you won't be able to do a `send('', {embed: this.loadList()})`